### ISSUE TITLE
arepo idealized galaxy stellar ages don't have a unit

### ISSUE THREAD

**Issue by dnarayanan** [yt-project/yt#2433]
if you run an idealized galaxy simulation with arepo, the stellar ages return as `dimensionless` (this is in yt4.x).   the age is actually given by:

```
 All.UnitTime = All.UnitLength / All.UnitVelocity
```

where this information is in the HDF5 header

```   
 ATTRIBUTE "UnitLength_in_cm" {DATATYPE  H5T_IEEE_F64LE
         DATASPACE  SCALAR
      }
      ATTRIBUTE "UnitMass_in_g" {
         DATATYPE  H5T_IEEE_F64LE
         DATASPACE  SCALAR
      }
      ATTRIBUTE "UnitVelocity_in_cm_per_s" {
         DATATYPE  H5T_IEEE_F64LE
         DATASPACE  SCALAR
      }
```

**Comment by ngoldbaum** [comment]
I think you just need to add an entry to this tuple for the field you're loading (not sure what the name is based on the info in the issue description):

https://github.com/yt-project/yt/blob/816301c2d511b4ea0ddac292004876a72bbf7bbe/yt/frontends/arepo/fields.py#L16-L31

**Comment by munkm** [comment]
Resolved via yt-project/yt#2434
